pytket/extensions/quantinuum/backends/api_wrappers.py

```diff
@@ -469,3 +470,130 @@ def cancel(self, job_id: str) -> dict:
         jr = res.json()
 
         return jr  # type: ignore
+
+
+class QuantinuumAPIOffline:
+    def __init__(
+        self,
+        machine_list: Optional[list] = [
+            {
+                "name": "H1-1",
+                "n_qubits": 20,
+                "gateset": ["RZZ", "Riswap", "Rxxyyzz"],
+                "n_classical_registers": 50,
+                "n_shots": 10000,
+                "system_family": "H1",
+                "system_type": "hardware",
+                "emulator": "H1-1E",
+                "syntax_checker": "H1-1SC",
+                "batching": True,
+                "wasm": True,
+            },
+            {
+                "name": "H1-2",
+                "n_qubits": 12,
+                "gateset": ["RZZ", "Riswap", "Rxxyyzz"],
+                "n_classical_registers": 50,
+                "n_shots": 10000,
+                "system_family": "H1",
+                "system_type": "hardware",
+                "emulator": "H1-2E",
+                "syntax_checker": "H1-2SC",
+                "batching": True,
+                "wasm": True,
+            },
+        ],
+    ):
+        """Initialize OfflineQuantinuum API client.
+        Trys to allow all the operations like the QuantinuumAPI without
+        any interaction with the remote device
+        All jobs that are submitted to this offline api are stored
+        and can be requested again later
+
+        :param machine_list: gives the parameter of all the devices
```

Comment:
To be useful, this docstring should explain the format required, or at least give an example.

Reply:
Maybe the current default parameter could just be moved to the docstring as an example?

Reply:
The QuantinuumAPIOffline will not work without this information, this will be set in case the list given is None. I have added a shorter example in the docstring to explain that a bit more.